src/components/course/CourseAssociatedPrograms.jsx

```diff
@@ -28,7 +28,7 @@ export default function CourseAssociatedPrograms() {
               </div>
             </div>
             <div className="col">
-              <a href={program.marketingUrl}>
+              <a href={program.marketingUrl} target="_blank" rel="noopener noreferrer">
```

Comment:
Should we use the `Hyperlink` component from Paragon? By providing `target="_blank"`, the component will automatically add the `rel` attribute and also add the external link icon to indicate it will open in a new tab.

https://paragon-edx.netlify.app/components/hyperlink/